script/add-codeql-to-path

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -eu
+
+cd "$(dirname "$0")/.."
```

Comment:
Not sure if we needed this 🙈  I just copied these lines from the other scripts 😂

Reply:
:grin: The purpose is to get you to the root of the repository, so that it doesn't matter what directory you're in when you run the script. There are lots of ways of achieving the same thing, but this is one of them.

In Bash the arguments that you pass on the command line are stored in `$1`, $2`, etc... and `$0` is used for the path to the script itself. That'll be something like `/home/rneatherway/qc-run2/script/add-codeql-to-path`, so then `dirname` will strip it back to `/home/rneatherway/qc-run2/script` and when you add the `..` it's effectively ``/home/rneatherway/qc-run2`.